snippets/csharp/VS_Snippets_CLR_System/system.Console.WriteLine/CS/WriteLine7.cs

```diff
@@ -1,15 +1,14 @@
-﻿// <Snippet7>
-using System;
+﻿using System;
 
 public class Example
 {
    public static void Main()
    {
+      // <Snippet7>
```

Comment:
let's undo this change since DateTime.Now doesn't pick up the user's local time so it looks a bit weird to have a date different than what you're expecting.

Reply:
Done, thanks.